test/action.yml

```diff
@@ -56,13 +46,18 @@ runs:
       run: |
         echo "local_snap=${{github.workspace}}/${{inputs.snap}}" >> $GITHUB_OUTPUT
 
+    # Checkout Repository with LFS
+    - uses: actions/checkout@v4
+      with:
+        repository: MonicaisHer/matter-snap-testing
+        ref: IENG-881
```

Comment:
Can't these be omitted and taken from defaults, via Github variables?

Reply:
Is this referring to `${{github.workspace}}`?

Reply:
These two have been updated now. `github.action_ref` has been documented in [github contexts]( https://docs.github.com/en/actions/learn-github-actions/contexts#github-context).

Reply:
According to the [github action log](https://github.com/canonical/chip-tool-snap/actions/runs/7502983667/job/20484717645#step:2:35), the `github.action_repository` has been set to `chip-tool-snap` instead of `matter-snap-testing`.

Reply:
Another workaround would be to extract them in a step and then pass to the action via environment variables. As previously done for action_path: https://github.com/MonicaisHer/matter-snap-testing/blob/4ec7094bdbca9b21578133011f03e7713b3704e4/test/action.yml#L78-L92